README.md

```diff
@@ -3,14 +3,19 @@ Plugin for checking a local Mailcow instance via the Mailcow API using Icinga2 /
 # Usage
 
 * **-d / --domain** The domain and optional port to reach the Mailcow instance
+* **-e / --endpoint** The endpoint to check (status/containers, status/solr, status/vmail)
```

Comment:
it might be easier to require containers / solr / vmail without status/. This would also require a slight code change.

Reply:
I thought about that. But I would like to also query e.g. "mailq/all" which doesn't use the status path.
If we want to allow that, we shouldn't hardcode "status" at this point IMHO.

Reply:
In this case, you are absolutely right. Do you intend to add this too at a later point?